test/BUILD.gn

```diff
@@ -76,8 +74,19 @@ static_library("brave_test_support_unit") {
 }
 
 if (!is_ios) {
-test("brave_unit_tests") {
+
+group("brave_unit_tests") {
```

Comment:
looks good to me, but just to educate myself: why is the split to group and test needed here?

Reply:
because we want the install name fix to run as part of the tests, not as a separate call to ninja https://github.com/brave/brave-browser/pull/9679/files#diff-dd2e881af3311f834199c7b515cff0c7L58 so the actual tests were renamed to *_impl